.github/readmes/javascript/_setup-2.md

```diff
@@ -20,6 +20,39 @@ prisma2 lift up
 
 You can now use the [SQLite Browser](https://sqlitebrowser.org/) to view and edit your data in the `./prisma/dev.db` file that was created when you ran `prisma2 lift up`.
 
+<Details>
+<Summary><b>Alternative: </b>Connect to your own database</Summary>
+
+Prisma supports MySQL and PostgreSQL at the moment. If you would like to connect to your own database, you can do so by specifying a different data source in the [Prisma schema file](prisma/schema.prisma).
+
+For a MySQL provider:
+```
+datasource mysql {
+    provider = "mysql"
+    url      = "mysql://johndoe:secret42@localhost:3306/mydatabase"
+}
+```
+
+*OR*
+
+For a PostgreSQL provider:
+```
+datasource postgresql {
+  provider = "postgresql"
+  url      = "postgresql://johndoe:secret42@localhost:5432/mydatabase?schema=schema.prisma"
```

Comment:
The `schema` argument inside the connection string should be the name of the PostgreSQL schema targeted (e.g. `public`, `myschema`, ...). `schema.prisma` is the file name of the Prisma schema file and doesn't have to do anything with the PostgreSQL schema. So much _schema ambiguity_ 🙈

Reply:
Eeeek..... no I should I have caught that!